apps/website/src/tailwind.css

```
@@ -1,66 +1,146 @@
+  themes:
+    light --default,
+    dark --prefersdark;
 }
 
-body {
-	--sl-color-bg-sidebar: #0c0c0e;
-	--sl-color-hairline-shade: #2d2d2d;
-	--sl-color-bg-nav: #0c0c0e;
-	@apply bg-base-100;
+@plugin "daisyui/theme" {
+  name: "light";
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)